infra-templates/ipsec/13/rancher-compose.yml

```diff
@@ -2,6 +2,7 @@
   name: "Rancher IPsec"
   version: "0.2.0"
   minimum_rancher_version: v1.6.11-rc1
+  maximum_rancher_version: v1.6.11-rc1
```

Comment:
@deniseschannon should it have an rc appended? That would make a template unavailable to v1.6.11 users, just wanted to make sure that's an intent.